docs/discover/kuery.asciidoc

```diff
@@ -0,0 +1,110 @@
+[[kuery-query]]
+=== Kuery
+
+experimental[This functionality is experimental and may be changed or removed completely in a future release.]
+
+Kuery is a new query language built specifically for Kibana. It aims to simplify the search experience in Kibana
+and enable the creation of helpful features like auto-complete, seamless migration of saved searches, additional
```

Comment:
It might be nice to add additional clarification that these helpful features (like autocomplete) aren't available now, but will be in the future.